css/style.css

```diff
@@ -0,0 +1,80 @@
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+main {
+    height: 100vh;
+    background-image: linear-gradient( 115deg, rgba(58, 58, 158, 0.8), rgba(136, 136, 206, 0.7) ),url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg);
+    padding: 3%;
+    background-size:cover;
+    background-repeat: no-repeat;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+}
+
+.heading {
+    color: white;
+}
+
+#title {
+    font-family: Georgia, 'Times New Roman', Times, serif;
```

Comment:
- [ ] The heading has a default font-weight, please make sure the weight (and font-family) matches the design from the template.